blocks/collapse/collapse.css

```diff
@@ -1,12 +1,82 @@
-.text-block__expandable-area {
+.section {
```

Comment:
Since this style and the ones below for different screen sizes are applicable beyond the `collapse` block, probably they should be moved to [styles.css](https://github.com/hlxsites/zeiss/blob/00cf136512cd2938aae1c06a38281ed023710bc5/styles/styles.css)